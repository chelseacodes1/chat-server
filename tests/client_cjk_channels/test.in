REGISTER ユー 猎人
LOGIN ユー 猎人
JOIN 
CREATE
JOIN 茶馆
CREATE 茶馆
JOIN 茶馆
CREATE 茶馆
SAY 茶馆 也许以后还有缘相见。