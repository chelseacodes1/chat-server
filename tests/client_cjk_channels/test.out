RESULT REGISTER 1
RESULT LOGIN 1
Error: Invalid use of JOIN command
Error: Invalid use of CREATE command
RESULT JOIN 茶馆 0
RESULT CREATE 茶馆 1
RESULT JOIN 茶馆 1
RESULT CREATE 茶馆 0
RECV ユー 茶馆 也许以后还有缘相见。
